tests/promela/test_model.py

```diff
@@ -98,6 +98,47 @@ def test_do():
     generate_and_verify(model, "do.pml")
 
 
+def test_atomic_alternative():
```

Comment:
No test for eventually?

Reply:
I can add some integration tests for this. I don't want to stick to a "valid" form of never claim, I anticipate many changes in this area, so integration test (executing spin on the model) should good.

Reply:
Integration tests added